packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -183,9 +197,9 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     ) public {
         // The realizedLPFeePct should never be greater than 0.5e18 and the slow and instant relay fees
```

Comment:
Why should `realizedLPFeePct < 0.5`?

Reply:
The reasoning is we constrain the slow and instant fees to each be less than 0.25e18, summing to less than 0.5e18. By definition, the `realizedLpFeePct` cant exceed 0.5e18 or the total fee percentage will exceed 1 (meaning that the total due in fees is more than the relayed amount). This require to prevent this situation.

Reply:
>By definition, the realizedLpFeePct cant exceed 0.5e18 or the total fee percentage will exceed 1 (meaning that the total due in fees is more than the relayed amount)

Why is this by definition? Can you explain in the comment, this part isn't that clear to me.